apis/userapi/userapi.yaml

```diff
@@ -1669,8 +1668,68 @@ paths:
         CollectParameters: false
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
-        IsMultiContentStreaming: false      
- 
+        IsMultiContentStreaming: false 
+        
+ /user/v1/tnc/accept:
+    post:
+      description: >-
+       This API is used to mark user acceptance of terms and conditions.
+          - The **_/user/v1/tnc/accept_** endpoint executes the "Accept T&C Api" request based on parameters provided as metadata in the request body
+          - It is mandatory to provide values for required parameters
+      summary: Accept Terms and Conditions
+      tags:
+      - User APIs
+      operationId: AcceptTnc
+      produces:
+      - application/json
+      parameters:
+      - name: Body
+        in: body
+        required: true
+        description: >-
+          - The body contains metadata about user's acceptance of given version of terms and conditions
+        schema:
+          allOf:
+              - $ref: '#/definitions/AcceptTnc'
+      - name: Content-Type
+        in: header
+        required: true
+        type: string
+        description: >- 
+          'Type of the request data. Set as application/json.'
+      - name: X-Authenticated-User-Token
+        in: header
+        required: true
+        type: string
+        description: >-
+          Access token of registered user performing given API request.
+      - name: ts
+        in: header
+        required: false
+        type: string
+        description: 'Timestamp at which given API request is sent.'
+      - name: X-msgid
+        in: header
+        required: false
+        type: string
+        description: 'This ID uniquely identifies a request if the same API is executed multiple times.'    
+          
+      responses:
+        '200':
+          description: 'OK. Successful operation. The requested operation was successfully executed.'
+          schema:
+            $ref: '#/definitions/CommonSuccessResponse'
+        '400':
+          description: >-
+           CLIENT_ERROR. The requested operation failed due to bad request from client. Possible reasons for failure: 
+            - Invalid value for parameter version
+            - Mandatory parameter version is missing.
+          schema:
+            $ref: '#/definitions/UserErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR. The requested operation failed due to a server error. '

```

Comment:
INTERNAL SERVER ERROR, we are tracking the error and cater to it on priority. Try to refresh the page and if the problem persists contact us at info@sunbird.org